src/Manuals/Shortcuts.blp

```diff
@@ -0,0 +1,72 @@
+using Gtk 4.0;
+
+ShortcutsWindow shortcuts-window {
+  modal: true;
+  hide-on-close: true;
```

Comment:
I removed this but when you try to open Manuals through the button, close it and repoen it you get this warning
```
A window is shown after it has been destroyed. This will leave the window in an inconsistent state.
```
and then you are unable to close the shortcuts window. But adding ``hide-on-close: true`` back again solves the problem.